config/v1/types_infrastructure.go

```diff
@@ -745,7 +744,6 @@ type NutanixPrismElementEndpoint struct {
 	Name string `json:"name"`
 
 	// endpoint holds the endpoint address and port data of the Prism Element (cluster).
-	// When a cluster-wide proxy is installed, this endpoint will not be accessed via the proxy.
```

Comment:
I would rather we update the comment than remove it

```suggestion
// When a cluster-wide proxy is installed, by default, this endpoint will be accessed via the proxy.
// Should you wish for communication with this endpoint not to be proxied, please add the endpoint to the
// proxy spec.noProxy list.
// +kubebuilder:validation:Required
```

Reply:
Changed as suggested.